### ISSUE TITLE
PDF/A-1B clause 6.5.3 test 4

### ISSUE THREAD

**Issue by a20god** [veraPDF/veraPDF-validation-profiles#146]
The description of test 4 for clause 6.5.3 of PDF/A-1B says:

"For all annotation dictionaries containing an AP key, the appearance dictionary that it defines as its value shall contain only the N key. If an annotation dictionary's Subtype key has a value of Widget and its FT key has a  value of Btn, the value of the N key shall be an appearance subdictionary; otherwise the value of the N key shall be an appearance stream."

However, that's clause 6.3.3 of ISO-19005-2:2011. Clause 6.5.3 of ISO-19005-1:2005 says:

"If an annotation dictionary contains the AP key, the appearance dictionary that it defines as its value shall
contain only the N key, whose value shall be a stream defining the appearance of the annotation."

In other words, that test confuses PDF/A-2 with of PDF/A-1.

**Comment by bdoubrov** [comment]
You are right that the original ISO 19005-1:2005 does not include this extra clause about button fields. However, this was added in its Corrigendum 2: http://www.iso.org/iso/home/store/catalogue_tc/catalogue_detail.htm?csnumber=60603

Our PDF/A-1 validation profile takes both Corrigenda 1 and 2 into account resulting in this particuar formulation and the rule. If someone needs to have strict ISO 19005-1:2005 validation without Corrigenda, this would require (slight) modifications of the published veraPDF PDF/A-1 profile.
